warehouse/query-core/src/test/java/datawave/query/testframework/GroupingAccumuloWriter.java

```diff
@@ -114,10 +114,10 @@ private void writeMetaData(BatchWriterConfig bwConfig, final List<Map.Entry<Mult
                         Value colVal = new Value(SummingCombiner.VAR_LEN_ENCODER.encode((long) rawData.get(column).size()));
                         mut.put(ColumnFamilyConstants.COLF_F, new Text(this.dataType + NULL_SEP + shardDate), colVal);
                         if (this.fieldConfig.getIndexFields().contains(column)) {
-                            mut.put(ColumnFamilyConstants.COLF_I, dtText, EMPTY_VALUE);
+                            mut.put(ColumnFamilyConstants.COLF_I, new Text(this.dataType + NULL_SEP + shardDate), colVal);
```

Comment:
Shouldn't these tests still work without the addition of the counts for i, ri for backward compatibility?

Reply:
hmm.... maybe.  I will look into in

Reply:
I jumped the gun when doing this. After reverting these changes, all tests still pass with the original data format.

Reply:
I would still like to see the tests without the addition of the new format to prevent us from accidentally breaking the old format handling in future.

Reply:
Reverted remaining changes to test data format. I believe I caught everything and that all tests should be passing with the original test data format.